### ISSUE TITLE
[Bug][v3-next] plurals is not a function

### ISSUE THREAD

**Issue by tienpham94** [lingui/js-lingui#705]
**Describe the bug**
When upgrading to @lingui/react v3/next, when using `{Plural or plural} from "@lingui/macro"` I have an error: `plurals is not a function`
![image](https://user-images.githubusercontent.com/25751050/83871448-bed72780-a738-11ea-9591-a53dd2c17434.png)
I have made a demo for the bug using the example/create-react-app in next branch as a PR
https://github.com/lingui/js-lingui/pull/704
My setup is exactly like the demo I made
```
"@lingui/react": "^3.0.0-13",
"@lingui/cli": "^3.0.0-13",
"@lingui/loader": "^3.0.0-13",
"@lingui/macro": "^3.0.0-13"
```

Added "macros" to .babelrc
```
{
  "plugins": [
    "macros"
  ]
}
```

**To Reproduce**
Set up @lingui v3 based on the following guides
https://js-lingui-git-next.lingui-js.now.sh/tutorials/setup-react.html
https://js-lingui-git-next.lingui-js.now.sh/releases/migration-3.html

Add localeData manually
```
import {i18n} from '@lingui/core';
import { en, ru } from 'make-plural/plurals'
i18n.loadLocaleData('en', { plurals: en })
i18n.loadLocaleData('ru', { plurals: ru })
```

Then use `i18n._(plural())` or `<Plural>`

```
import { Plural, plural } from "@lingui/macro"
import { useLingui } from "@lingui/react"

// Using either `i18n._(plural())` or `<Plural>` will break with error: `plurals is not a function`
export default function App() {
  const { i18n } = useLingui()
   return (
     <p>
          {i18n._(
            plural(2, {
              one: "You have # friend",
              other: "You have # friends",
            })
          )}
          <Plural
            value={friends}
            one="You have # friend"
            other="You have # friends"
          />
      </p>
    )
}
```
**Added localeData manually but the error still persists, like the demo**
=> Error `plurals is not a function` appears and the app crashes

**Expected behavior**
`<Plural>` and `i18n._(plural())` should work correctly

- jsLingui version 3.0.0-13
- Babel version "@babel/core": "^7.8.7",
- Your Babel config (e.g. `.babelrc`): I use Create React App, if not, I will have this in .babelrc
```
{
  "plugins": [
    "macros"
  ]
}
```

**Comment by lucasecdb** [comment]
duplicate of #683

**Comment by tienpham94** [comment]
I added localeData manually but the error still persists
```
import {i18n} from '@lingui/core';
import { en, ru } from 'make-plural/plurals'
i18n.loadLocaleData('en', { plurals: en })
i18n.loadLocaleData('ru', { plurals: ru })
```

**Comment by tricoder42** [comment]
This should be fixed in `3.0.0-18`. It happend during initial render before messages were loaded.
